presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfigurationInitializer.java

```diff
@@ -66,13 +66,13 @@
     private int textMaxLineLength;
 
     @VisibleForTesting
-    public HdfsConfigurationUpdater(HiveClientConfig config)
+    public HdfsConfigurationInitializer(HiveClientConfig config)
     {
         this(config, ignored -> {});
     }
 
     @Inject
-    public HdfsConfigurationUpdater(HiveClientConfig config, S3ConfigurationUpdater s3ConfigurationUpdater)
+    public HdfsConfigurationInitializer(HiveClientConfig config, S3ConfigurationUpdater s3ConfigurationUpdater)
```

Comment:
Should `S3ConfigurationUpdater` become a `DynamicConfigurationProvider`?

Reply:
I thought about that as well, but it has complications, so let's defer that.